corehq/apps/saved_reports/tasks.py

```diff
@@ -145,20 +147,29 @@ def send_email_report(self, recipient_emails, domain, report_slug, report_type,
 
     subject = cleaned_data['subject'] or _("Email report from CommCare HQ")
 
-    content = _render_report_configs(
-        mock_request, [config], domain, user_id, couch_user, True, lang=couch_user.language,
-        notes=cleaned_data['notes'], once=once
-    )[0]
-    body = render_full_report_notification(None, content).content
-
     try:
+        content = _render_report_configs(
+            mock_request, [config], domain, user_id, couch_user, True, lang=couch_user.language,
+            notes=cleaned_data['notes'], once=once
+        )[0]
+        body = render_full_report_notification(None, content).content
+
         for recipient in recipient_emails:
             send_HTML_email(subject, recipient,
                             body, email_from=settings.DEFAULT_FROM_EMAIL,
                             smtp_exception_skip_list=LARGE_FILE_SIZE_ERROR_CODES)
 
     except Exception as er:
-        if getattr(er, 'smtp_code', None) in LARGE_FILE_SIZE_ERROR_CODES:
+        notify_exception(
+            None,
+            message="Encountered error while sending email",
+            details={
+                'subject': subject,
+                'recipients': str(recipient_emails),
+                'error': er,
+            }
+        )
+        if getattr(er, 'smtp_code', None) in LARGE_FILE_SIZE_ERROR_CODES or type(er) == ESError:
             # If the smtp server rejects the email because of its large size.
             # Then sends the report download link in the email.
```

Comment:
Can you update this comment?

Reply:
https://github.com/dimagi/commcare-hq/pull/23782/commits/813d9daea54cdd41706bc657cde6dcfb31db2d02